server/routes/index.ts

```diff
@@ -23,6 +23,7 @@ require('./dashboardCollectionLayout');
 
 /* Routes for PubPub */
 require('./communityCreate'); // Route: '/community/create'
+require('./communityServices'); // Route: '/community/create'
```

Comment:
ultra-super-nit: fix the route comment

Reply:
💯 I'm all for the ultra-super-nits. Nice catch!